README.md

```diff
@@ -167,6 +167,30 @@ export default function App() {
 }
 ```
 
+### Get the access token
+
+Sometimes it is useful to obtain the access token directly (e.g. to make API requests to another service).
+
+```jsx
+import { getAccessToken } from '@workos-inc/authkit-nextjs';
+
+export default async function CoolComponent() {
+  const accessToken = await getAccessToken();
+
+  if (!accessToken) {
```

Comment:
it's kind of annoying to have to wonder if you're getting an access token back or not. i wonder if `accessToken` should instead (or additionally?) be another piece of data returned from `getUser` (that way you choose the `ensureSignedIn` behavior).

Reply:
I'm leaning towards `accessToken` being a part of the data returned from `getUser`. 

Eventually we might want to rename `getUser` to something like `getAuth` to make it clearer you can do this:

```js
const { accessToken, user } = await getAuth();
```